package.json

```diff
@@ -142,10 +141,10 @@
     "ts-node": "^2.0.0",
     "tslint": "~4.3.1",
     "typedoc": "^0.5.3",
-    "typescript": "2.0.10",
+    "typescript": "~2.1.5",
```

Comment:
Landed in 2.4.2

![screen shot 2017-01-12 at 3 38 56 pm](https://cloud.githubusercontent.com/assets/8420490/21909397/5d97fd14-d8dd-11e6-884d-6610f1793f6a.png)